src/components/editor-page/editor-pane/linter/frontmatter-linter.spec.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * SPDX-FileCopyrightText: 2022 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+import { mockI18n } from '../../../markdown-renderer/test-utils/mock-i18n'
+import type { Diagnostic } from '@codemirror/lint'
+import { mockEditorView } from './single-line-regex-linter.spec'
+import { FrontmatterLinter } from './frontmatter-linter'
+import { t } from 'i18next'
+
+const testFrontmatterLinter = (content: string, diagnostics: Partial<Diagnostic>, replace?: string): void => {
+  const frontmatterLinter = new FrontmatterLinter()
+  const editorView = mockEditorView(content)
+  const spy = jest.spyOn(editorView, 'dispatch')
```

Comment:
Optional: You could move the spy into the if-statement because it's only used there.